src/main/java/nextstep/subway/domain/Distance.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.subway.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+    @Column
+    private int distance;
+
+    protected Distance() {
+    }
+
+    public Distance(int distance) {
+        validate(distance);
+        this.distance = distance;
+    }
+
+    private void validate(int distance) {
+        if (distance <= 0) {
+            throw new IllegalArgumentException("거리는 0보다 작을 수 없습니다.");
+        }
+    }
+
+    public int getDistance() {
```

Comment:
```suggestion
    public int value() {
```

내부 변수가 한개 이므로 `value`로 바라보아도 좋을것? 같네요 
개인적인 생각입니다!